src/App.js

```diff
@@ -1,10 +1,18 @@
-import React from "react";
+import React, {useState} from "react";
+import {Route} from "react-router-dom";
+import Home from "./Components/Home"
+import Header from "./Components/Header"
+import Pizza from "./Components/Pizza"
+import Confirmation from "./Components/Confirmation"
 
 const App = () => {
+  const [order, setOrder] = useState({})
   return (
     <>
-      <h1>Lambda Eats</h1>
-      <p>You can remove this code and create your own header</p>
+    <Header />
+    <Route path='/' exact component={Home}/>
```

Comment:
one of the newer features in react-router is the ability to use a component like 
`<Route path='/'><Home></Route>`

Nothing wrong with how you did it here, but it simplifies things when you need to pass props.